Paintroid/src/main/java/org/catrobat/paintroid/MainActivity.java

```diff
@@ -119,10 +122,10 @@ public void onCreate(Bundle savedInstanceState) {
 		setTheme(R.style.PocketPaintTheme);
 		super.onCreate(savedInstanceState);
 
-		PaintroidApplication.cacheDir = getCacheDir();
-		PaintroidApplication.checkeredBackgroundBitmap = BitmapFactory.decodeResource(getResources(), R.drawable
-				.pocketpaint_checkeredbg);
-
+		getAppFragment();
```

Comment:
This should not be here as well I think. The changes in these classes are from another pull request.